src/main/java/prefs/UpdateConfig.java

```diff
@@ -0,0 +1,35 @@
+package prefs;
+
+import util.Version;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Contains configuration information for updating
+ */
+public class UpdateConfig {
+
+    private boolean lastUpdateDownloadStatus;
+    private List<Version> listOfVersionsPreviouslyDownloaded; // NOPMD
```

Comment:
The warning is that it can be made `final`, but if it is final, gson can't do conversion. Should I add a comment about it?